pkg/controller/drain.go

```diff
@@ -179,7 +185,23 @@ func (o *DrainOptions) RunDrain() error {
 		return err
 	}
 
-	err := o.deleteOrEvictPodsSimple()
+	node, err := o.client.CoreV1().Nodes().Get(o.nodeName, metav1.GetOptions{})
```

Comment:
Another question: does the logic belong here? Perhaps, it should be moved higher up into `machine.go`?

Reply:
I don't have a strong opinion if this should be here or outside. I would be fine keeping it here.